austation/code/datums/mutations/telekinesis.dm

```diff
@@ -0,0 +1,3 @@
+/mob/living/carbon/attack_tk(mob/living/carbon/owner)
+	src.help_shake_act(owner)
```

Comment:
You need . = ..() here if the parent proc should be called

Reply:
Was this tested? With various items to ensure previous functionality works

Reply:
It was tested. Since carbons don't inheret any attack_tk methods, I left it as this.